### DISCUSSION TITLE
后台用户权限设置里面关于角色的设置

### DISCUSSION THREAD

**Discussion by gianni2009** [yangjian102621/chatgpt-plus#86] [规划中的任务]
### ⚠️  确认 issue 是否已存在 ⚠️

- [X] 我已经搜索了现有的问题，没有找到跟我问题相关的问题。

### GPT-3 or GPT-4

GPT-4

### 操作系统

Windows

### Bug 描述 📝

1、后台给某一个或某一类比如vip，设置了可以使用的角色的权限，但是前台还是能看到全部的角色和使用角色。并不能限制使用的角色范围。
2、后台的角色希望添加一个分类的选项，比如：工作类，生活，编程等可以自定义。前台一堆的角色太乱。

![微信图片_20231228140204](https://github.com/yangjian102621/chatgpt-plus/assets/153248944/1a1a9483-7168-4149-92e3-4cdfd7a5e251)

### 你的应用配置信息

```toml
把你的配置信息粘贴到这里
```

**Comment by yangjian102621** [comment]
收到，感谢反馈，已将此 issue 转入规划需求中。
